lib/matplotlib/transforms.py

```diff
@@ -682,6 +682,77 @@ def intersection(bbox1, bbox2):
 class Bbox(BboxBase):
     """
     A mutable bounding box.
+
+    Examples
+    --------
+    **Create from known bounds**
+
+    The default constructor takes the boundary "points" ``[[xmin, ymin],
+    [xmax, ymax]]``.
+
+        >>> Bbox([[1, 1], [3, 7]])
+        Bbox([[1.0, 1.0], [3.0, 7.0]])
+
+    Alternatively, a Bbox can be created from the flattened array, the
+    so-called "extents" ``(xmin, ymin, xmax, ymax)``
+
+        >>> Bbox.from_extents(1, 1, 3, 7)
+        Bbox([[1.0, 1.0], [3.0, 7.0]])
+
+    or from the "bounds" ``(xmin, ymin, width, height)``.
+
+        >>> Bbox.from_bounds(1, 1, 2, 6)
+        Bbox([[1.0, 1.0], [3.0, 7.0]])
+
+    **Create from collections of points**
+
+    The "empty" object for accumulating Bboxs is the null bbox, which has
```

Comment:
This is more „properties of the null box“ and might be better moved there. While you are using the null box in the following example, the null boxes dimensions are not relevant there. Without knowing what a null box is, you can still „create a null box and update it with the points“.

Reply:
It could be moved there, but I think it makes sense for the user to see "why" the null Bbox is the "empty" object. I mean they can always read past this information if it's not relevant for them?

Reply:
My update is a compromise that doesn't emphasize the properties of the null Bbox until later, but still prints it, so that the user can infer why it is "like" the empty set if they "feel" like thinking that hard.

Reply:
If you feel strongly that the new version still feels out-of-order, I'll change it :)